src/components/Category/Category.jsx

```diff
@@ -1,20 +1,35 @@
-import React from "react";
+import axios from "axios";
+import React, { useState, useEffect } from "react";
 import { Link } from "react-router-dom";
-import { categories } from "../../staticData";
 import "./Category.css";
 
 export const Category = () => {
+	const [categoryList, setCategorList] = useState([]);
+
+	useEffect(() => {
+		try {
+			(async () => {
+				const {
+					data: { categories },
+				} = await axios.get("/api/categories");
+				setCategorList(categories);
+			})();
+		} catch (error) {
+			console.log(error);

```

Comment:
The error should reflect in the ui as well, so the user also knows whats going on.

Reply:
Okay, I will change that.